sample.ts

```diff
@@ -0,0 +1,52 @@
+import fc from "fast-check";
+
+/**
+ * Script function to compare two fast-check generators by sampling them.
+ * @param generatorA The first generator to compare.
+ * @param generatorB The second generator to compare.
+ * @param numSamples The number of samples to compare.
+ * @returns An array of result objects containing the sample number, values
+ * generated by each generator, and the match object containing whether the
+ * values match and the mismatched items.
+ */
+export const compareGenerators = (
+  generatorA: fc.Arbitrary<any>,
+  generatorB: fc.Arbitrary<any>,
+  numSamples: number
+) => {
+  const results = [];
+  for (let i = 0; i < numSamples; i++) {
+    const seed = 123 * i;
```

Comment:
Since this test suite now uses fast-check, pass the seed as a parameter (same we did for the number of sample).

Then we can always set the global seed with fc.configureGlobal in the test suite root for replays.

Reply:
Turns out this whole file is not needed at all. See https://github.com/stacks-network/rendezvous/pull/69#discussion_r1894582703.